examples/units/evans_demo.py

```diff
@@ -1,8 +1,10 @@
 """
 ==========
-Evans test
+Evans demo
```

Comment:
I don't think this is an example either. It could possibly become a tutorial "How to use custom units", but certainly needs more explanation for that. Btw. does somebody know why it's called "Evans test"?